src/core/lib/gprpp/load_file.cc

```diff
@@ -32,7 +32,8 @@ namespace grpc_core {
 
 // Loads the content of a file into a slice. add_null_terminator will add a NULL
 // terminator if true.
-absl::StatusOr<Slice> LoadFile(std::string filename, bool add_null_terminator) {
+absl::StatusOr<Slice> LoadFile(const std::string& filename,
```

Comment:
The exact reason for using `std::string` here is because the implementation does need it as a null-terminated string.  If we take in `std::string`, then we know it's null-terminated; if we take in `absl::string_view`, then we have no such guarantee.

To say this another way, if we changed this to take `absl::string_view`, then the implementation would have to use `std::string(filename).c_str()` when passing it to `fopen()`, so we'd wind up unconditionally making a copy.  In contrast, by taking in `const std::string&`, we still wind up making a copy if the caller passes in a `char*`, but at least we can avoid that copy if the caller happens to already have a `std::string`.